<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookupTableEditWidget</class>
 <widget class="QWidget" name="LookupTableEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colormap Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Value range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autoCheckBox">
        <property name="text">
         <string>Automatic</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="RealNumberEditWidget" name="maxEdit">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="fillUpperCheckBox">
          <property name="text">
           <string>Fill upper area</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="minLabel">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="RealNumberEditWidget" name="minEdit">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="fillLowerCheckBox">
          <property name="text">
           <string>Fill lower area</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="divisionNumberLabel">
          <property name="text">
           <string>Division Number:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="maxLabel">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="divisionNumberSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colormap</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ColorMapSettingWidget" name="colorMapSetting" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorMapSettingWidget</class>
   <extends>QWidget</extends>
   <header>guibase/colormap/colormapsettingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
